addons/website_event/views/event_event_views.xml

```diff
@@ -47,6 +47,69 @@
+                        <tree>
+                            <field name="sequence" widget="handle" />
+                            <field name="title"/>
+                            <field name="is_mandatory_answer" string="Mandatory"/>
+                            <field name="once_per_order" string="Once per Order"/>
+                            <field name="question_type" string="Type" />
+                            <field name="answer_ids" widget="many2many_tags"
+                                invisible="question_type != 'simple_choice'" />
+                            <button name="action_view_question_answers" type="object" class="p-0" icon="fa-bar-chart pe-1" string="Stats"
+                                    title="Answer Breakdown" invisible="question_type not in ['simple_choice', 'text_box']"/>
+                        </tree>
+                        <!-- Need to repeat the whole tree form here to be able to create answers properly
+                            Without this, the sub-fields of answer_ids are unknown to the web framework.
+                            We need this because we create questions and answers when the event type changes. -->
+                        <form string="Question">
+                            <sheet>
+                                <h1 class="d-flex"><field name="title" placeholder='e.g. "Do you have any diet restrictions?"' class="flex-grow-1"/></h1>
+                                <group>
+                                    <group>
+                                        <field name="is_mandatory_answer"/>
+                                        <field name="question_type" widget="radio"/>
+                                    </group>
+                                    <group>
+                                        <field name="once_per_order"/>
+                                    </group>
+                                </group>
+                                <notebook invisible="question_type != 'simple_choice'">
+                                    <page string="Answers" name="answers">
+                                        <field name="answer_ids">
+                                            <tree editable="bottom">
+                                                <!-- 'display_name' is necessary for the many2many_tags to work on the event view -->
+                                                <field name="display_name" column_invisible="True" />
+                                                <field name="sequence" widget="handle" />
+                                                <field name="name"/>
+                                            </tree>
+                                        </field>
+                                    </page>
+                                </notebook>
+                            </sheet>
+                        </form>
+                    </field>
+                </page>
+            </page>
```

Comment:
This will duplicate the question page, that was moved to event module a while back, we can remove it I believe?

Reply:
Oh, I didn't see that, that's not my code. Probably a bad rebase or conflict resolution. I'll look into it.